elastic/elastic-agent
# Osquerybeat: Fix osquerybeat is not running with logstash output

## What does this PR do?

Enables all beats supported outputs including logstash.
The issue was that the agent was not running osquerybeat for the policy with logstash output due to outdated spec. This updates osquerybeat spec adding all supported outputs.

Addresses 
[Asset Mgt] [Bug] "matching app is not found for action input: osquery" error while running osqueries with Agent Policy having "Output" type in Fleet as "Logstash(beta)"

https://github.com/elastic/security-team/issues/4216


## Why is it important?

Addresses 
[Asset Mgt] [Bug] "matching app is not found for action input: osquery" error while running osqueries with Agent Policy having "Output" type in Fleet as "Logstash(beta)"

https://github.com/elastic/security-team/issues/4216

## Checklist


## How to test this PR locally

1. Configure the policy with logstash output
2. Add osquery manager to that policy
3. Run the agent
4. Ensure that osquerybeat is running and you can send live queries for example

## Related issues

- Closes https://github.com/elastic/security-team/issues/4216

## Screenshots

Osquerybeat is running for the policy with logstash output
<img width="2186" alt="Screen Shot 2022-07-05 at 9 23 36 PM" src="https://user-images.githubusercontent.com/872351/177448198-db13b7d9-380f-4139-be2a-8a9750e2335e.png">


## Logs
<img width="2222" alt="Screen Shot 2022-07-05 at 9 23 49 PM" src="https://user-images.githubusercontent.com/872351/177448219-74571a08-8d14-4c07-8849-f8fe71109051.png">
